linux-tkg-config/prepare

```diff
@@ -304,7 +304,7 @@ _set_cpu_scheduler() {
   elif [ "$_kver" = "606" ]; then
     _avail_cpu_scheds=("eevdf" "bore")
   else
-    _avail_cpu_scheds=("cfs")
+    _avail_cpu_scheds=("cfs" "eevdf")
```

Comment:
actually here, if I remember well, this block for the newest rc kernels, so you can simply put `eevdf` alone.